components/fragment_renderer/fragment_components/ImageWithCollapse.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import ImageWithCollapse from "./ImageWithCollapse";
+import { Default } from "./ImageWithCollapse.stories.js";
+import { axe, toHaveNoViolations } from "jest-axe";
+import { userEvent } from "../../../node_modules/@storybook/test/dist/index";
+
+expect.extend(toHaveNoViolations);
+
+describe("ImageWithCollapse", () => {
+  test("renders ImageWithCollapse component with default props", async () => {
+    const { container } = render(<ImageWithCollapse {...Default.args} />);
+    expect(screen.getByAltText("image alt text")).toBeInTheDocument();
+    expect(
+      screen.getByText((content) => content.startsWith("Every"))
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText((content) => content.startsWith("Example Title"))
+    ).toBeInTheDocument();
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
+
+  test("on click renders Collapse component with default props", async () => {
+    const { container } = render(<ImageWithCollapse {...Default.args} />);
+    const collapse = screen.getByText((content) =>
+      content.startsWith("Example Title")
+    );
+    userEvent.click(collapse);
```

Comment:
Instead of just clicking on it and checking for the strings (the strings are accessible by the testing library whether open or closed), is there a way to check that the Collapse is opened after the click? It uses a `<details>` html tag so it might have something built in for a11y that could be leveraged.

Reply:
I'll look into it!